src/mgmt/tests/srv_drpc_tests.c

```diff
@@ -1568,6 +1598,55 @@ test_drpc_exclude_bad_uuid(void **state)
 	D_FREE(resp.body.data);
 }
 
+static void
+test_drpc_exclude_mgmt_svc_fails(void **state)
+{
+	Drpc__Call	call = DRPC__CALL__INIT;
+	Drpc__Response	resp = DRPC__RESPONSE__INIT;
+
+	setup_exclude_drpc_call(&call, TEST_UUID, 0);
+	ds_mgmt_exclude_return = -DER_UNKNOWN;
+
+	ds_mgmt_drpc_pool_exclude(&call, &resp);
+	expect_drpc_exclude_resp_with_error(&resp,
+					       ds_mgmt_exclude_return);
+
+	D_FREE(call.body.data);
+	D_FREE(resp.body.data);
+}
+
+static void
+test_drpc_exclude_invalid_rank(void **state)
```

Comment:
Rank isn't checked in the dRPC handler, but deeper inside the mgmt svc instead, right? So this unit test is testing the same logic as the mgmt_svc_fails one above - it gets down to the mock and gets an error back. Would be a valid thing to check at the system level, but in the mocked unit tests it isn't testing any new code paths. So I think this test is redundant, and should be removed to avoid confusion.